kolibri/core/exams/models.py

```diff
@@ -158,6 +158,10 @@ class Meta:
     """
     data_model_version = models.SmallIntegerField(default=3)
 
+    # If True, learners have instant access to exam reports after submission.
+    # Otherwise, reports are visible only after the coach ends the exam.
+    instant_report_visibility = models.BooleanField(default=True)
```

Comment:
I think we may want to be careful about exactly how we add this field - my specific thought is in relation to the migration this will produce on KDP.

When we add a Django model field with a default value, this can result in a very long running migration, if the table is large.

The way to avoid this is to allow the field to be nullable `null=True` - and handle the null values, and return them as `True` via the API.

We create the migration with `null=True` and no default. Then we add the `default=True` and create another migration. At this point, we can merge the two migration files into one, just as long as the AddField operations are first, and then the AlterField to add the default comes after.